Frontend/android-native/app/src/main/java/in/juspay/mobility/utils/OverlaySheetService.java

```diff
@@ -144,6 +144,18 @@ public void onViewHolderBind(SheetAdapter.SheetViewHolder holder, int position,
             holder.destinationAddress.setText(model.getDestinationAddress());
             holder.textIncPrice.setText(String.valueOf(model.getNegotiationUnit()));
             holder.textDecPrice.setText(String.valueOf(model.getNegotiationUnit()));
+            if(model.getSourcePinCode() != null &&  model.getSourcePinCode().trim().length()>0){
+                holder.sourcePinCode.setText(model.getSourcePinCode().trim());
+                holder.sourcePinCode.setVisibility(View.VISIBLE);
+            }else{
+                holder.sourceAddress.setMaxLines(2);
+            }
+            if(model.getDestinationPinCode() != null && model.getDestinationPinCode().trim().length()>0){
+                holder.destinationPinCode.setText(model.getDestinationPinCode());
+                holder.destinationPinCode.setVisibility(View.VISIBLE);
+            }else{
+                holder.destinationAddress.setMaxLines(2);
```

Comment:
`holder.destinationPinCode.setVisibility(View.GONE);`
Add this line in else case